components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -6820,24 +6891,14 @@ func insertTestProject(
 	id string,
 	name string,
 	projType storage.Type) storage.Project {
-	return insertTestProjectWithRules(t, db, id, name, projType, storage.NoRules)
-}
-
-func insertTestProjectWithRules(
```

Comment:
This function was misleading because it let you pass in a status but it simply returned a project with no rules. Dropped the function and explicitly set the rule status expected in the expect part of the tests that were using it.